### PR TITLE
Kelsonic 7791 checkbox yr

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/7791

This was a weird one. The input field has `position: absolute;` and it had no immediate parent that had `position: relative;`, which made it so that it _appeared_ as if you couldn't toggle the checkbox directly.

This PR fixes it by adding custom styling. I'd prefer not to add custom styling, but alas, it did resolve the problem for now. We may want to consider fixing this in the design system.

This PR also includes the content tweaks in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7806) and [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7805).

## Testing done
No tests needed updating.

#### Before
image

#### Resolved
image

## Acceptance criteria
- [x] Make content updates.
- [x] Fix click area for checkboxes on both mobile and desktop.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/yellow-ribbon/sass/yellow-ribbon.scss [hunk 1]
```diff
@@ -2,8 +2,20 @@
...
   outline: none;
 }
+
+.form-checkbox {
+  position: relative;
+
+  & [type="checkbox"] {
+    cursor: pointer;
+  }
+}
```
Comment (ncksllvn) [inline src/applications/yellow-ribbon/sass/yellow-ribbon.scss:21]: I'm still confused why we need any custom CSS. This structure doesn't work for YR? https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/user/profile/vet360/containers/ReceiveTextMessages.jsx#L96

Comment (kelsonic) [reply src/applications/yellow-ribbon/sass/yellow-ribbon.scss:21]: Oh nice! Let me see if adding that wrapping div works like they did!

Comment (kelsonic) [reply src/applications/yellow-ribbon/sass/yellow-ribbon.scss:21]: Hey! That worked! 🎉

Comment (kelsonic) [reply src/applications/yellow-ribbon/sass/yellow-ribbon.scss]: All we needed was just a wrapping `div` actually

File: src/applications/yellow-ribbon/containers/SearchForm/index.jsx [hunk 2]
```diff
@@ -185,21 +185,23 @@ export class SearchForm extends Component {
...
-        />
-
-        {/* Unlimited Number of Students */}
-        <ErrorableCheckbox
-          checked={numberOfStudents === 'unlimited'}
-          label="Only show schools that provide funding to all eligible students"
-          onValueChange={onCheckboxChange('numberOfStudents')}
-          required={false}
-        />
+        <div>
```
Comment (ncksllvn) [inline src/applications/yellow-ribbon/containers/SearchForm/index.jsx:188]: I don't know what it does, but does this need `className="form-checkbox-buttons"`?

Comment (ncksllvn) [reply src/applications/yellow-ribbon/containers/SearchForm/index.jsx:188]: That's what I see at https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/user/profile/vet360/containers/ReceiveTextMessages.jsx#L96

Comment (kelsonic) [reply src/applications/yellow-ribbon/containers/SearchForm/index.jsx:188]: It doesn't, `form-checkbox-buttons` has no styles applied to it lol

**Review by ncksllvn — APPROVED** [review]

**Pull request merged**